### ISSUE TITLE
Extensions: "group" should provide intellisense to show valid groups to contribute menus to

### ISSUE THREAD

**Issue by bpasero** [Microsoft/vscode#42475]
![image](https://user-images.githubusercontent.com/900690/35570021-2f201ad8-05ce-11e8-8cdf-c7903449d14b.png)

/cc @jrieken

**Comment by jrieken** [comment]
They are kind of open-ended

**Comment by bpasero** [comment]
Yeah, but maybe our core ones could be in there. I have often seen people ask about a list for groups that exist (same as when conditions btw).
